_docs/conversions.md

```diff
@@ -0,0 +1,113 @@
+It should look something like this:
+
+```
+dockstoreToken=abcdefghijklmnopqrstuvwxyz1234567890
+server-url=https://www.dockstore.org/api
+organization=test_organization
+repo=test_repository
+write-api-url=http://localhost:8080/api/ga4gh/v1
+```
+
+"dockstoreToken" is acquired from your account page on the dockstore website .
+"server-url" is the dockstore server url.
+"organization" is the organization/user of the repository to create.
+"repo" is the repository to create.
+"write-api-url" is the url of the write-api-service
+
+### Server Configuration
+The write-api-service uses githubToken and quayToken from the server yml file and looks something like this:
+
+```
+quayioToken: abcdefghijklmnopqrstuvwxyz1234567890
+githubToken: abcdefghijklmnopqrstuvwxyz1234567890
+...
+```
+The github token can be attained by following the instructions from [GitHub Help](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/)
+The quay.io token can be attained by following the instruction from [Quay.io API docs](https://docs.quay.io/api/) -> OAuth 2 Access Tokens -> Generating a Token (for internal application use).  When generating the token, provide these 3 permissions:
+* Create Repositories
+* View all visible repositories
+* Read/Write to any accessible repositories
+
+### Client Usage
+```
+Usage: client [options] [command] [command options]
+  Options:
+    --help
+      Prints help for the client.
+      Default: false
+  Commands:
+    add      Add the Dockerfile and CWL file(s) using the write API.
+      Usage: add [options]
+        Options:
+        * --Dockerfile
+            The Dockerfile to upload
+        * --cwl-file
+            The cwl descriptor to upload
+          --cwl-secondary-file
+            The optional secondary cwl descriptor to upload
+          --help
+            Prints help for the add command
+            Default: false
+          --version
+            The version of the tool to upload to
+
+    publish      Publish tool to dockstore using the output of the 'add'
+            command.
+      Usage: publish [options]
+        Options:
+          --help
+            Prints help for the publish command.
+            Default: false
+        * --tool
+            The json output from the 'add' command.
+```
+
+### Sample Client Output
+```
+client add --Dockerfile Dockerfile --cwl-file Dockstore.cwl --cwl-secondary-file Dockstore2.cwl --version 3.0
+{
+  "githubURL": "https://github.com/dockstore-testing/test_repo3",
+  "quayioURL": "https://quay.io/repository/dockstore-testing/test_repo3",
+  "version": "3.0"
+}
+```
+You can pipe the output like this:
+```
+client add --Dockerfile Dockerfile --cwl-file Dockstore.cwl --cwl-secondary-file Dockstore2.cwl --version 3.0 > test.json
+```
+and then:
+```
+client publish --tool test.json
+```
+
+
+## Converting File-path Based Imports to Public http(s) Based Imports
```

Comment:
This section doesn't really belong here (it's not really related to the write API above)
I'm not sure where it belongs, maybe https://docs.dockstore.org/docs/user-tutorials/language-support/ but open to ideas. 

Watch out for links like at the bottom of https://docs.dockstore.org/docs/user-tutorials/firecloud-launch-with/